hypertrace-view-generator/hypertrace-view-generator-api/src/main/avro/SpanEventView.avdl

```diff
@@ -48,7 +48,7 @@ protocol SpanEventViewProtocol {
     long duration_millis = 0;
 
     //this will be >=0 only for spans with API_BOUNDARY_TYPE = ENTRY
-    long internal_duration_millis = -1;
+    union { null, long } internal_duration_millis = null;
```

Comment:
this change doesn't need any tests to assert?

Reply:
Not sure what exactly it is checking, but there's a github action to validate avros: `build and validate / validate-avros (push)`. The build has passed it.

Reply:
`SpanEventViewGeneratorTest` doesnt have any such test?
 asked since the test for `RawServiceViewGeneratorTest` was added for similar addition

Reply:
It already exists: https://github.com/razorpay/hypertrace-ingester/blob/10db5874e04a14c5a6d333584401915f9437fe30/hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/SpanEventViewGeneratorTest.java#L326